azurerm_application_gateway/outputs.tf

```diff
@@ -0,0 +1,4 @@
+# output "azurerm_application_gateway_id" {
```

Comment:
Pleease, do not commit commented code. Let's maybe put the app gateway name for now (or id)?